.github/workflows/ci.yaml

```diff
@@ -0,0 +1,186 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: .nvmrc
+          cache: npm
+
+      - name: Install dependencies
```

Comment:
Can we forego `name` throughout? We're accustomed to seeing commands in CI, and the name tends to hide that info under a section that has to be expanded to view. And similar to CCI, GH Actions that we depend on will provide their own.

Reply:
Much better! ✅ 

![Screenshot 2024-06-18 at 12 16 31 PM](https://github.com/zendeskgarden/react-components/assets/6879688/95edb732-0fab-4338-a036-a6809858cb8b)